<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlterarProduto</class>
 <widget class="QDialog" name="AlterarProduto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>208</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>100</horstretch>
    <verstretch>100</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>9</width>
    <height>8</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Alterar Produto - QEstoqueLoja</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">

#AlterarProduto{
background-color: rgb(199, 199, 199);

}
QPushButton{
background-color: rgb(43, 132, 191);
color: rgb(255, 255, 255);
font-size: 15px;
border-radius: 13px;

}
QPushButton:disabled{
	
	
	background-color: rgb(154, 153, 150);
}

QPushButton:hover{
	background-color: rgb(13, 92, 161);
}
QPushButton:pressed{
	
color: rgb(255, 255, 255);
	background-color: rgb(153, 193, 241);
font-size: 15px;
border-radius: 13px;
}
QGroupBox{
background-color:rgb(170, 170, 170);
	font: 700 9pt &quot;Segoe UI&quot;;
	font-size: 15px;
}
</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Código de Barras:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Ledit_AltBarras"/>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_GerarCod">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip">
        <string extracomment="Gerar Código de Barras Pessoal">Gerar Código de Barras Pessoal</string>
       </property>
       <property name="statusTip">
        <string extracomment="Gerar Código de Barras Pessoal">Gerar Código de Barras Pessoal</string>
       </property>
       <property name="accessibleDescription">
        <string>Gerar Código de Barras Pessoal</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Descrição: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Ledit_AltDesc"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Quantidade: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Ledit_AltQuant"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Preço (R$): </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Ledit_AltPreco"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="Check_AltNf">
     <property name="text">
      <string>NF</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="Btn_AltAceitar">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Btn_AltAceitar</sender>
   <signal>accepted()</signal>
   <receiver>AlterarProduto</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_AltAceitar</sender>
   <signal>rejected()</signal>
   <receiver>AlterarProduto</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
